packages/server/scripts/test.sh

```diff
@@ -5,8 +5,8 @@ if [[ -n $CI ]]
 then
   # --runInBand performs better in ci where resources are limited
   export NODE_OPTIONS="--max-old-space-size=4096"
-  echo "jest --coverage --runInBand --forceExit"
-  jest --coverage --runInBand --forceExit
+  echo "jest --coverage --forceExit"
+  jest --coverage --forceExit
```

Comment:
Will the changes in this pr make a difference such that we no longer need to runInBand in CI?

Reply:
I did split the test to run both worker and server separately (as they are the flaky ones). With this change, it looks like the `runInBand` is not required (at least they are not failing) and hopefully will speed the tests a little bit

Reply:
Just FYI, the `runInBand` was much faster from previous testing. It can help in environments with limited resources. 

Could be worth trying a before and after commit in this pr just to see

Reply:
I checked and adding it runs in a similar time. I will add it back to avoid further issues until we fix the overall issue with the tests being slow